docs/DeveloperGuide.adoc

```diff
@@ -277,106 +277,365 @@ Refer to the guide <<DevOps#, here>>.
 [appendix]
 == Product Scope
 
-*Target user profile*:
+**Target user profile:**
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+  - Has a need to manage multiple trips
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+  - Prefers using a notebook to other types
+
+  - Frequently uses the computer while overseas
+
+  - Wants to micromanage all parts of their trips
+
+  - Wants to plan all details of the trip before leaving
+
+  - Wants to manage a trip even without an internet connection
+
+**Value proposition:** Able to micromanage a trip and access one’s plans
+more conveniently than traditional forms of trip planning
 
 [appendix]
 == User Stories
 
-Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
+image::us1.PNG[]
+image::us2.PNG[]
+image::us3.PNG[]
+image::us4.PNG[]
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
-|=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+[appendix]
+== Use Cases
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+**Use case: UC1 - Add Trip**
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+***MSS***
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+1.  User requests to **Trip Manager** to list trips
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+2.  TravelPal shows a list of **Trips**
 
-_{More to be added}_
+3.  User requests to add a specific **Trip** to the list
 
-[appendix]
-== Use Cases
+4.  User <span class="underline">edits the **Trip** (UC2)</span>
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+5.  TravelPal adds the **Trip**
 
-[discrete]
-=== Use case: Delete person
+6.  TravelPal shows the list of **Trips**
+    
+    Use case ends.
```

Comment:
The "Use case ends." formatting is different for each use case, should be consistent